js/test/components/operatorSignIn/operatorSignInModal.test.tsx

```diff
@@ -161,7 +176,6 @@ describe("OperatorSignInModal", () => {
     await userEvent.type(view.getByRole("textbox"), "123");
     await userEvent.click(view.getByRole("button", { name: "OK" }));
 
-    expect(view.getByText("Step 2 of 2")).toBeInTheDocument();
```

Comment:
Why are these assertions removed?

Reply:
Per [the task AC](https://app.asana.com/0/1206105669438487/1208729753338210/f) we're removing the "Step 2 of 2" language